src/transaction/log_replication_jobs.cpp

```diff
@@ -57,12 +62,11 @@ namespace cublog
 	const int64_t end_time_msec = util_get_time_as_ms_since_epoch ();
 	const int64_t time_diff_msec = end_time_msec - a_start_time_msec;
 
-	// assert (time_diff_msec >= 0);
 	if (time_diff_msec > 0)
 	  {
 	    perfmon_set_stat (thread_p, PSTAT_REDO_REPL_DELAY, static_cast<int> (time_diff_msec), false);
 
-	    if (prm_get_bool_value (PRM_ID_ER_LOG_CALC_REPL_DELAY))
+	    if (output_debug_log)
 	      {
 		_er_log_debug (ARG_FILE_LINE, "[CALC_REPL_DELAY]: %lld msec", time_diff_msec);
 	      }
```

Comment:
How about just using er_log_deaug instead? It prints only if `prm_get_bool_value (PRM_ID_ER_LOG_DEBUG)`.
```suggestion
	    if (prm_get_bool_value (PRM_ID_ER_LOG_CALC_REPL_DELAY))
	      {
		er_log_debug (ARG_FILE_LINE, "[CALC_REPL_DELAY]: %lld msec", time_diff_msec);
	      }
```

Reply:
`PRM_ID_ER_LOG_DEBUG` is already accounted for upon initialization of `output_debug_log`.

Reply:
I suggested using `er_log_debug` without `output_debug_log` to write logging only in the debug mode. This function seems suited for your purpose. It's up to you.